src.ts/account.ts

```diff
@@ -51,15 +51,16 @@ export class Account {
     }
 
     async fetchState(): Promise<void> {
+        this._accessKey = null;
         this._state = await this.connection.provider.query(`account/${this.accountId}`, '');
-        try {
-            const publicKey = (await this.connection.signer.getPublicKey(this.accountId, this.connection.networkId)).toString();
-            this._accessKey = await this.connection.provider.query(`access_key/${this.accountId}/${publicKey}`, '');
-            if (this._accessKey === null) {
-                throw new Error(`Failed to fetch access key for '${this.accountId}' with public key ${publicKey}`);
-            }
-        } catch {
-            this._accessKey = null;
+        const publicKey = await this.connection.signer.getPublicKey(this.accountId, this.connection.networkId);
+        if (publicKey === null) {
```

Comment:
@ilblackdragon I'd prefer we use `!publicKey` here and in other cases when checking for absence of value (which cannot be a zero number). Otherwise even `undefined` won't pass this check.